Azure/azure-cli
# [keyvault] fix: --include-pending requires true or false

Fix #11121: When using `az keyvault certificate list`, passing `--include-pending` now doesn't require a value of `true` or `false`, similar to `--enable-soft-delete`.

To test:
```
az keyvault certificate list --vault-name {} --include-pending
```
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).
